### PR TITLE
Feature/synthesis

### PR DESCRIPTION
**Pull request by JeanRochCoulon**
Add gate simulation based on netlist produced by synthesis done from cva6 repository. Update cva6 hash.

### CODE REVIEW

**Comment by MikeOpenHWGroup** [comment]
It all looks good to me @JeanRochCoulon, but I see that you have marked this as a draft, so I will wait until that is lifted to approve.

**Comment by JeanRochCoulon** [comment]
Hello @MikeOpenHWGroup, this PR allows to make gate simulations, good to verify synthesis result. The goal of the pointed CVA6 hash was to be able to synthetize for ASIC.

**Review by MikeOpenHWGroup — APPROVED** [review]

**Pull request merged**
